packages/perseus-editor/src/components/locked-point-settings.tsx

```diff
@@ -68,105 +78,194 @@ const LockedPointSettings = (props: Props) => {
         setCoordState(newCoord);
     }
 
+    function handleColorChange(newValue) {
+        onChangeProps({color: newValue});
+    }
+
     return (
-        <View style={styles.container}>
-            {/* Title row */}
-            <View style={styles.row}>
-                <LabelLarge>Point</LabelLarge>
-                <Spring />
-                <IconButton
-                    icon={trashIcon}
-                    aria-label={`Delete locked point at ${coordState[0]}, ${coordState[1]}`}
-                    onClick={onRemove}
-                />
-            </View>
-
-            {/* Coordinates */}
-            <View>
-                <View style={styles.row}>
-                    <LabelMedium
-                        htmlFor={xCoordId}
-                        style={styles.label}
-                        tag="label"
-                    >
-                        x Coordinate
-                    </LabelMedium>
-                    <TextField
-                        id={xCoordId}
-                        value={coordState[0]}
-                        onChange={(newValue) => handleCoordChange(newValue, 0)}
-                        onBlur={handleBlur}
-                        style={styles.textField}
-                    />
-                </View>
+        <View
+            // More specificity so that we can override the default
+            // universal styles from the .less file.
+            className="locked-figure-accordion"
+        >
+            <AccordionSection
```

Comment:
I do agree that it would be awesome to break this down. I think I might favor a different approach myself - I think I'll break out the color circle and the color select into different components, and that should make this significantly easier to follow. And then they can be reused for locked lines!

Reply:
I'll give that a try and if you still feel like it should be split up by accordion, header, and panel, I'll go ahead and do that